File: be/server/src/main/java/com/baseball/domain/InningInfo.java
package com.baseball.domain;

public class InningInfo {

    private final Integer inningCount;
    private final Boolean isUserTop;
    private final Boolean isUserOffense;

    public InningInfo(Integer inningCount, Boolean isUserTop, Boolean isUserOffense) {
        this.inningCount = inningCount;
        this.isUserTop = isUserTop;
        this.isUserOffense = isUserOffense;
    }

    public Integer getInningCount() {
        return inningCount;
    }

    public Boolean getUserTop() {
        return isUserTop;
    }

    public Boolean getUserOffense() {
        return isUserOffense;
    }
}

File: ios/baseball/baseball/Domain/Entity/InningInfo.swift
//
//  InningInfo.swift
//  baseball
//
//  Created by zombietux on 2021/05/10.
//

import Foundation

struct InningInfo: Codable {
    private(set) var inningCount: Int
    private(set) var userTop: Bool
    private(set) var userOffense: Bool
    
    func isUserTop() -> String {
        return userTop ? "초" : "말"
    }
    
    func isUserOffense() -> String {
        return userOffense ? "공격" : "수비"
    }
}
